cirq-core/cirq/testing/deprecation.py

```diff
@@ -11,14 +11,18 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+import contextlib
 import logging
 import os
-from typing import Optional
+from typing import Iterator, Optional
+
+import cirq.testing.logs as logs
 
```

Comment:
CI is barfing on this.  Think we use `import cirq.testing` then `cirq.testing.asser_logs` for method call below?

Reply:
Reverted to use inline import.